bin/testObservability/helper/helper.js

```diff
@@ -475,14 +484,18 @@ exports.batchAndPostEvents = async (eventUrl, kind, data) => {
   };
 
   try {
+    const eventsUuids = data.map(eventData => `${eventData.event_type}:${eventData.test_run ? eventData.test_run.uuid : (eventData.hook_run ? eventData.hook_run.uuid : null)}`).join(', ');
+    exports.debugOnConsole(`[Request Batch Send] for events:uuids ${eventsUuids}`);
     const response = await nodeRequest('POST',eventUrl,data,config);
+    exports.debugOnConsole(`[Request Batch Response] for events:uuids ${eventsUuids}`);
     if(response.data.error) {
       throw({message: response.data.error});
     } else {
       exports.debug(`${kind} event successfull!`)
```

Comment:
let's add a log here as well, if success

Reply:
Logging the complete response. If there is no error then it's successful. Do we need a redundant log?